File: patterns/c-patterns/pattern121.c
// Pattern 121: ...

// ***
// ***
// **
// **
// *
// *

#include <stdio.h>

int main()
{
    int i, j, x, y;

    for (i = 3; i >= 1; i--)
    {
        for (x = 2; x > 0; x--)
        {
            for (j = 0; j < i; j++)
            {
                printf("*");
            }
            printf("\n");
        }
    }
    return 0;
}

File: patterns/javascript-patterns/pattern121.js
// Pattern 121: ...

// ***
// ***
// **
// **
// *
// *

for (let i = 3; i >= 1; i--) {
    for (let x = 2; x > 0; x--) {
        let stars = '';
        for (let j = 0; j < i; j++) {
            stars += '*';
        }
        console.log(stars);
    }
}
